src/modules/flow/persistence/Kconfig

```diff
@@ -0,0 +1,3 @@
+config FLOW_NODE_TYPE_PERSISTENCE
+	tristate "Node type: persistence"
+	default m
```

Comment:
depends on filesystem or efivars being available, no? Or at least linux at this time.